cmd/cmd.go

```diff
@@ -227,6 +228,18 @@ func (o *options) sternConfig() (*stern.Config, error) {
 			t = "{{.Message}}"
 		case "json":
 			t = "{{json .}}"
+		case "extjson":
+			t = "\"pod\": \"{{color .PodColor .PodName}}\", \"container\": \"{{color .ContainerColor .ContainerName}}\", \"msg\": {{extjson .Message}}"
```

Comment:
I want to change `msg` to `message`. It should not be abbreviated.

```suggestion
			t = "\"pod\": \"{{color .PodColor .PodName}}\", \"container\": \"{{color .ContainerColor .ContainerName}}\", \"message\": {{extjson .Message}}"
```